phylogenetic/rules/export.smk

```diff
@@ -42,7 +42,7 @@ rule prepare_auspice_config:
     output:
         auspice_config="results/config/{gene}/auspice_config_{serotype}.json",
     params:
-        replace_clade_key="clade_membership",
+        replace_clade_key=lambda wildcard: r"clade_membership" if wildcard.gene in ['genome'] else r"nextclade_subtype",
         replace_clade_title=lambda wildcard: r"Serotype" if wildcard.serotype in ['all'] else r"DENV genotype",
```

Comment:
To mirror the title `NCBI serotype` added below, maybe the default `clade_membership` title should be updated to `Nextstrain serotype`?

Reply:
Thank you for flagging! I'm considering renaming `NCBI serotype` to`Serotype (NCBI)` to better match naming convention used in [measle's `Genotype (NCBI)` title](https://next.nextstrain.org/measles/genome?c=genotype_ncbi). Then, I agree that the default title for `clade_membership` should be renamed from `Serotype` to `Serotype (Nextstrain)`.

To recap:

* `NCBI serotype` -> `Serotype (NCBI)`: indicating that denv1-4 assignment is based on NCBI GenBank record annotation
* `Serotype` -> `Serotype (Nextstrain)`: indicating that denv1-4 assignment is based on augur clades call using full-genome-level-serotype-defining amino acid mutations
* `Nextclade genotype` -> `Genotype (Nextclade)`: indicating genotype level assignment within the serotype (e.g. `DENV1/S`) based on Nextclade call

This naming adjustment leaves space for a potential `Genotype (NCBI)` if we develop a script for parsing genotype annotations from the GenBank data.

Reply:
Ah, actually I think I'll merge this pull request as it is, and handle the renaming mentioned above in a later PR. That way, we can address it along with fixing this issue: https://github.com/nextstrain/dengue/issues/41.